diff --git a/node_modules/react-native-code-highlighter/src/lib/CodeHighlighter.tsx b/node_modules/react-native-code-highlighter/src/lib/CodeHighlighter.tsx
index d034ada..42aaedf 100644
--- a/node_modules/react-native-code-highlighter/src/lib/CodeHighlighter.tsx
+++ b/node_modules/react-native-code-highlighter/src/lib/CodeHighlighter.tsx
@@ -1,4 +1,4 @@
-import React, { type FunctionComponent, type ReactNode, useMemo } from "react";
+import { type FunctionComponent, type ReactNode, useMemo } from "react";
 import {
 	ScrollView,
 	type ScrollViewProps,
